include/simdjson/fallback/numberparsing.h

```diff
@@ -11,12 +11,13 @@ void found_float(double result, const uint8_t *buf);
 namespace simdjson {
 namespace SIMDJSON_IMPLEMENTATION {
 namespace {
+// credit: https://johnnylee-sde.github.io/Fast-numeric-string-to-int/
 static simdjson_really_inline uint32_t parse_eight_digits_unrolled(const char *chars) {
-  uint32_t result = 0;
-  for (int i=0;i<8;i++) {
-    result = result*10 + (chars[i] - '0');
-  }
-  return result;
+  uint64_t val;
+  memcpy(&val, chars, sizeof(uint64_t));
+  val = (val & 0x0F0F0F0F0F0F0F0F) * 2561 >> 8;
+  val = (val & 0x00FF00FF00FF00FF) * 6553601 >> 16;
+  return uint32_t((val & 0x0000FFFF0000FFFF) * 42949672960001 >> 32);
```

Comment:
Evidently, this code could be, in theory, slower on some 32-bit hardware than the original fallback, but it will almost surely be faster on any 64-bit processor.